pkg/distributor/distributor_test.go

```diff
@@ -374,6 +374,7 @@ func TestDistributor_MetricsCleanup(t *testing.T) {
 	d.nonHASamples.WithLabelValues("userA").Add(5)
 	d.dedupedSamples.WithLabelValues("userA", "cluster1").Inc() // We cannot clean this metric
 	d.latestSeenSampleTimestampPerUser.WithLabelValues("userA").Set(1111)
+	d.labelValuesWithNewlinesPerUser.WithLabelValues("userA").Inc()
```

Comment:
this change does nothing, unless we ask for new metric too (add `cortex_distributor_label_values_with_newlines_total` to `metrics`)

Reply:
Thanks, made the test meaningful in https://github.com/grafana/mimir/pull/9400/commits/0bed651811575e74e4d58d3c51c89b9720242ee6